validation/error.go

```diff
@@ -0,0 +1,5 @@
+package validation
+
+type Error struct {
```

Comment:
could this be compressed into a type alias?

`type Errors []string... `?

Reply:
it could be but the json marshal o of that would be an array (`[]`) instead of an object - you'd have to embed it somewhere anyway.